libbeat/docs/communitydashboards.asciidoc

```diff
@@ -0,0 +1,29 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content appears in both the Beats Platform Reference and the
+//// Beats Developer Guide.
+//////////////////////////////////////////////////////////////////////////
+
+[[community-dashboards]]
+== Community Beats
+
+The open source community has been hard at work developing new Dashboards. You can check
+out some of them here. For information on exporting and importing dashboards see the documentation for
+https://www.elastic.co/guide/en/kibana/current/managing-saved-objects.html[Kibana Saved Objects].
+
+Have a question about a community Dashboard? You can post questions and discuss issues in the
+https://discuss.elastic.co/c/beats/community-dashboards[Community Dashboards] category of the Beats discussion forum.
```

Comment:
You'll need to make sure this category exists before merging this PR, or the doc build will fail during the link checking phase.

Reply:
For sure. Thanks for the reminder